components/plugin-manager/implementation/build.gradle.kts

```diff
@@ -39,6 +39,8 @@ repositories {
 dependencies {
     api(projects.components.pluginManager.api)
 
+    implementation(projects.components.authorization.implementation)
```

Comment:
Hm, so it is allowed for components to depend on the implementation module of another component?